_config.yml

```diff
@@ -70,6 +70,16 @@ workshop_interactive_queues:
 # Start time in minutes (0 to be clock-independent, 540 to show a start at 09:00 am).
 start_time: 0
 
+####################################################
+# WORKSHOP specific configuration
```

Comment:
Recommend replacing `# WORKSHOP specific` with `# SITE specific` and removing `workshop_` from variable names. In the Markdown, variables are called using `{{ site.variable }}` Liquid syntax, which implies this is a *site*-specific setting. Having to type out `workshop_` seems both redundant and laborious to me.

Reply:
Excellent idea. I’ll implement that ASAP. Thanks for the review (as always).

Reply:
added with 21cc8ea